helpers/filterSeats.js

```diff
@@ -104,6 +106,152 @@ module.exports = function(dust) {
       });
     }
 
+    function reorderRates(rates, replyIndex) {
+      var bandTypes = {};
+      _.forEach(bandColours, function (sectionObj) {
+        _.forEach(sectionObj, function (quality) {
+          bandTypes[quality] = [];
+        });
+      });
```

Comment:
This section could do with a comment to explain where the data is coming from and what we're doing with it here